src/components/Banners/Implementations/WritersCohortBanner.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react"
+import { Text } from "@chakra-ui/react"
+
+import DismissableBanner from "../DismissableBanner"
+import Link from "../../Link"
+
+const WritersCohortBanner = () => {
+  if (
+    typeof window !== `undefined` &&
+    window.location.pathname.includes("contributing")
+  ) {
+    return (
+      <DismissableBanner storageKey="writersCohort">
+        <Text m={0}>
+          Join the first-ever ethereum.org Writers Cohort, in collaboration with
+          Taptive, starting May 11th.{" "}
```

Comment:
Can probably remove Taptive from here for simplicity. 

```suggestion
          Join the first-ever ethereum.org Writers Cohort, starting May 11th.{" "}
```